benchmarks/benchmark_gpu_sparsity.py

```diff
@@ -0,0 +1,248 @@
+import argparse
+import random
+
+import pandas as pd
+import torch
+import torch.utils.benchmark as benchmark
+import torch.nn.functional as F
+from torch import nn
+from torch.sparse import SparseSemiStructuredTensor, to_sparse_semi_structured
```

Comment:
does this require nightlies?

Reply:
We should probably log `torch.__version__`, but this doesn't require nightlies. Is there a way we can track torchao version as well?